deploy/Chart/templates/database/configmaps.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: database-secret
+  namespace: "{{ .Release.Namespace }}"
+  labels:
+    control-plane: database
+    app.kubernetes.io/name: database
+    app.kubernetes.io/part-of: radius
+data:
+  POSTGRES_DB: ps_db
+  POSTGRES_USER: ps_user
+  POSTGRES_PASSWORD: SecurePassword
```

Comment:
Note to reviewers, we're going to take this in steps. Right now the username/password are hardcoded.